main.py

```diff
@@ -2,6 +2,33 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create([
+        Genre(name="Western"),
+        Genre(name="Action"),
+        Genre(name="Drama"),
+    ])
+
+    Actor.objects.bulk_create([
+        Actor(first_name="George", last_name="Klooney"),
+        Actor(first_name="Kianu", last_name="Reeves"),
+        Actor(first_name="Scarlett", last_name="Keegan"),
+        Actor(first_name="Will", last_name="Smith"),
+        Actor(first_name="Jaden", last_name="Smith"),
+        Actor(first_name="Scarlett", last_name="Johansson"),
+    ])
+
+    Genre.objects.filter(name="Dramma")\
```

Comment:
```suggestion
    Genre.objects.filter(
        name="Dramma"
        ).update(name="Drama")
```

Reply:
change here and in all of the code

Reply:
I made changes, but flake8 is complaining about them."

Reply:
I made changes, but flake8 is complaining about them.